### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by Tenzer**
This fixes a number of smaller documentation issues:

1. The URLs in the README file were changed to reflect the current path to the repository.
2. The examples have been tested and corrected where there were changes. I also expanded the cache file example to make it a bit clearer how it might be used.
3. Finally, I regenerated the HTML files and made sure the docs dependencies accurately reflect which are needed for doing that.

### CODE REVIEW

**Comment by rec** [comment]
Wow, this is amazing, I haven't thought of this project for years!, I should have tried to publicize it more.

I'm just going to bed now, I'll check it out in the morning.

Thanks so much for doing this!!

File: README.rst [hunk 1]
```diff
@@ -50,20 +50,20 @@ Usage examples
...
-    #   /home/tom/.cache/my-project
+    #   /home/tom/.cache
 
     app.xdg.XDG_CONFIG_DIRS
     #   /etc/xdg
 
     with app.config.open() as f:
-        f.update(name='oliver', species='dog')
-        f['description'] = {'size': 'S', 'fur': 'brown'}
+        f.contents.update(name='oliver', species='dog')
```
Comment (rec) [inline README.rst:59]: Oof, this example was never right! :-)

**Review by rec — APPROVED** [review]
Well, this is just fine, and I have no changes to request.

I'm using pdoc3 elsewhere now, it's just a much better system.

Thanks so much, again, for doing this!!!

**Comment by Tenzer** [comment]
You're welcome :)

**Pull request merged**